submitted_models/costar_shafter_sensor_config_1/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(costar_shafter_sensor_config_1)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+install(DIRECTORY launch meshes urdf
```

```suggestion
install(DIRECTORY launch materials meshes urdf
```

Installing the `materials` directory as well seems to resolve the texture issue I mentioned [previously](https://github.com/osrf/subt/pull/632#pullrequestreview-558499286). However, even after resolving the texture not being found, the robot's frame still looks the same (it looks like the texture that was fixed is just for the LED light) - do you guys have any additional textures to add for the robot's frame/body?